src/components/FormComponents/ContactEditor.jsx

```diff
@@ -29,17 +33,40 @@ const ContactEditor = ({
   disabled,
   updateContact,
   updateContactEvent,
+  updateContactRor,
+  updateContactOrcid,
 }) => {
   const orgEmailValid = validateEmail(value.orgEmail);
   const indEmailValid = validateEmail(value.indEmail);
   const orgURLValid = validateURL(value.orgURL);
   const givenNamesValid = !value.givenNames?.includes(",");
   const lastNameValid = !value.lastName?.includes(",");
-  
+  const [rorInputValue, setRorInputValue] = useState(value.orgRor);
```

Comment:
@n-a-t-e I've tried to populate the initial state from the `value` object here - but it seems like this component is initially rendered before that `value` is available.
Is there a good way to set the state once it comes through?